.github/workflows/retest.yaml

```diff
@@ -1,14 +1,15 @@
 name: Detect and Trigger Retest
 on:
   issue_comment:
-    types: [created, edited]
+    types: [created]
 
 permissions:
   contents: read
 
 jobs:
   retest:
-    if: github.event.comment.author_association == 'MEMBER' && github.event.comment.body == '/retest'
+    # PR comments where a Member types "/retest" exactly
+    if: github.event.issue.pull_request && github.event.comment.author_association == 'MEMBER' && github.event.comment.body == '/retest'
```

Comment:
So, this only works for Members? Can I assume that Reviewers, Approvers, and Leads are all in the 'MEMBER' author_association?

Reply:
Yes, it's anyone in the Argoproj org. Note that we all also have the "Member" next to our name in comments.

Also that part hasn't changed in the PR 😅

Reply:
> Note that we all also have the "Member" next to our name in comments.
Got it!

> Also that part hasn't changed in the PR
Hey, I can still ask questions :)

Reply:
For sure! I think I missed that you approved first and then commented